README.md

```diff
@@ -123,6 +123,19 @@ Check two paths are equal or not. Trailing slash and encoding are normalized bef
 isSamePath('/foo', '/foo/')
 ```
 
+### `isURL`
+
+Check whether a given path has a protocol. To consider paths beginning with `//` as URLs, pass `true` as the second parameter (but note that [protocol-relative URLs are an anti-pattern](https://www.paulirish.com/2010/the-protocol-relative-url/)).
+
+```ts
+// Result: true
+isURL('https://nuxtjs.org/')
+// Result: false
+isURL('/')
+// Result: true
+isURL('//', true)
```

Comment:
`//` is not really a valid URL even considering scheme is optional since has no path:

```
URI = scheme:[//authority]path[?query][#fragment]
```

Reply:
Trying in chrome, `<a href="//>` redirects to `about:blank#blocked`